neqo-http3/src/connection.rs

```diff
@@ -1238,6 +1265,8 @@ impl Http3Connection {
         let stream_id = conn
             .stream_create(stream_type)
             .map_err(|e| Error::map_stream_create_errors(&e))?;
+        // set outgoing WebTransport streams to be fair (share bandwidth)
+        conn.stream_fairness(stream_id, true).ok();
```

Comment:
please don't ignore the return value, or at least don't do it this way (use .unwrap() explicitly)